examples/log-generator/lib/log-generator.js

```diff
@@ -0,0 +1,31 @@
+const axios = require('axios')
+const loremIpsum = require('lorem-ipsum')
+const { logger } = require('./logger')
+
+const request = axios.create({ baseURL: 'http://localhost:3000' })
+
+const logGenerator = (timeout) => { 
+  if (!timeout) throw new Error(
```

Comment:
You can set it a default one into Dockerfile too, to be more explicit for the image user that this variable could be changed